hapi/src/utils/consent2life.js

```diff
@@ -2,7 +2,7 @@ const crypto = require('crypto')
 
 const eosUtil = require('./eos')
 
-const CONTRACT_NAME = 'consent2life' // @todo: use ENV
+const CONTRACT_NAME = process.env.CONSENT2LIFE_CONTRACTNAME
```

Comment:
We are using a folder named config to keep all configurations in one place an import it from there https://github.com/eoscostarica/lifebank/tree/master/hapi/src/config